src/util/json.h

```diff
@@ -31,13 +31,22 @@ Convenience header for Nlohmann::Json that sets the default options. Also
 #include "util/Exception.h"
 #include "util/SourceLocation.h"
 
+// For higher flexibility of the custom json helper functions.
+// TODO Update with the new `SameAsAny`.
+template <typename T>
+concept IsOrderedOrUnorderedJson =
+    std::same_as<T, nlohmann::json> || std::same_as<T, nlohmann::ordered_json>;
+
 /*
 @brief Read the specified json file and build a json object out of it.
 
+@tparam JsonObjectType Either `nlohmann::json`, or `nlohmann::ordered_json`.
+
 @param jsonFileName Name of the file, or path to the file. Must describe a
 `.json` file.
 */
-inline nlohmann::json fileToJson(std::string_view jsonFileName) {
+template <IsOrderedOrUnorderedJson JsonObjectType>
+JsonObjectType fileToJson(std::string_view jsonFileName) {
```

Comment:
I think just `Json` suffices in this local context instead of `JsonObjectType`.

Reply:
Done, see [601d03c](https://github.com/ad-freiburg/qlever/pull/1221/commits/601d03c19d7ada930b31b38dc587a8a65da67f09).